dart/lib/src/sentry.dart

```diff
@@ -105,8 +105,8 @@ class Sentry {
     // execute integrations after hub being enabled
     if (appRunner != null) {
       var runIntegrationsAndAppRunner = () async {
-        final integrations = options.integrations
-            .where((i) => !(i is RunZonedGuardedIntegration));
+        final integrations =
```

Comment:
@ueman I know you don't like it, but I actually find it easier to read when there many `({})` around.

Reply:
Well, it's not my decision anymore 😅

Reply:
Yeah, we only use `is!` and `is`, maybe that's why it's easy for me, so they logically make sense.
If we start using `!is`, it'd be confusing.

Reply:
> Well, it's not my decision anymore 😅

Your opinion counts :)